sway/scene_descriptor.c

```diff
@@ -38,7 +38,9 @@ void *scene_descriptor_try_get(struct wlr_scene_node *node,
 
 void scene_descriptor_destroy(struct wlr_scene_node *node,
 		enum sway_scene_descriptor_type type) {
-	struct scene_descriptor *desc = scene_node_get_descriptor(node, type);
+	struct scene_descriptor *desc = scene_node_get_descriptor(node, type);	if (!desc) {
+		return;
```

Comment:
The formatting is off here. Please put a newline before the if.

Reply:
ok, done